core/src/test/java/org/infinispan/tx/totalorder/statetransfer/TotalOrderStateTransferFunctionalTest.java

```diff
@@ -63,4 +65,8 @@ protected void createCacheManagers() throws Throwable {
       configurationBuilder.memory().storageType(storage);
    }
 
+   @Override
+   public void testSTWithWritingNonTxThread(Method m) throws Exception {
+      super.testSTWithWritingNonTxThread(m);
+   }
 }
```

Comment:
it would be good to override all tests
```
   @Override
   public void testSTWithWritingNonTxThread(Method m) throws Exception {
      super.testSTWithWritingNonTxThread(m);
   }

   @Override
   public void testInitialStateTransfer(Method m) throws Exception {
      super.testInitialStateTransfer(m);
   }

   @Override
   public void testInitialStateTransferCacheNotPresent(Method m) throws Exception {
      super.testInitialStateTransferCacheNotPresent(m);
   }

   @Override
   public void testConcurrentStateTransfer(Method m) throws Exception {
      super.testConcurrentStateTransfer(m);
   }

   @Override
   public void testSTWithThirdWritingNonTxCache(Method m) throws Exception {
      super.testSTWithThirdWritingNonTxCache(m);
   }

   @Override
   public void testSTWithThirdWritingTxCache(Method m) throws Exception {
      super.testSTWithThirdWritingTxCache(m);
   }

   @Override
   public void testSTWithWritingTxThread(Method m) throws Exception {
      super.testSTWithWritingTxThread(m);
   }

   @Override
   public void testInitialStateTransferAfterRestart(Method m) throws Exception {
      super.testInitialStateTransferAfterRestart(m);
   }
```